Dockerfile

```diff
@@ -536,25 +561,24 @@ RUN set -x; \
 		# If this bug happens to trigger in the future, adding a "zypper -n download" of a subset of the packages can avoid the segfault.
 		zypper -n install bison>3.4 && \
 		# Allow fluent-bit to find systemd
-		ln -fs /usr/lib/systemd /lib/systemd && \
-		# Download and install CMake manually (zypper only gives us CMake 3.5, but we need >= 3.12)
-		wget https://github.com/Kitware/CMake/releases/download/v3.25.2/cmake-3.25.2-linux-x86_64.sh -O cmake.sh && \
-		(echo '4d98de8d605da676e71a889dd94f80c76abb377fade2f21e3510e62ece1e1ada  cmake.sh' | sha256sum --check) && \
-		bash cmake.sh --skip-license --prefix=/usr/local
-	
+		ln -fs /usr/lib/systemd /lib/systemd
 		ADD https://github.com/adoptium/temurin11-binaries/releases/download/jdk-11.0.13%2B8/OpenJDK11U-jdk_x64_linux_hotspot_11.0.13_8.tar.gz /tmp/OpenJDK11U-jdk_x64_linux_hotspot_11.0.13_8.tar.gz
```

Comment:
Just realized — don't you need to change this as well (`x64`)?

Reply:
Blah, yet another arch syntax to handle. It's extra painful because we don't need to support sles12 yet anyway. Might have to use the same strategy here that we used for CMake.

Reply:
Fixed.